src/EPR.RegulatorService.Facade.API/Controllers/RegulatorController.cs

```diff
@@ -159,7 +170,7 @@ public async Task<IActionResult> GetUserDetails()
             var userId = User.UserId();
             if (userId == Guid.Empty)
             {
-                _logger.LogError($"Unable to get the OId for the user when attempting to get organisation details");
+                _logger.LogInformation("Unable to get the OId for the user when attempting to get organisation details");
```

Comment:
this is was previously _logger.LogError any reson why it is now _logger.LogInformation?